parsec/core/cli/reencrypt_workspace.py

```diff
@@ -0,0 +1,46 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import click
+
+
```

Comment:
single newline between groups of imports

Reply:
A bit surprised this wasn't caught by the pre-commit, but it's fixed.